docs/docs/KerasStyleAPIGuide/Layers/advancedActivations.md

```diff
@@ -0,0 +1,332 @@
+---
+## **ELU**
+Exponential Linear Unit.
+
+It follows: f(x) =  alpha * (exp(x) - 1.) for x < 0, f(x) = x for x >= 0.
+
+**Scala:**
+```scala
+ELU(alpha = 1.0, inputShape = null)
+```
+**Python:**
+```python
+ELU(alpha=1.0, input_shape=None)
+```
+
+**Parameters:**
+
+* `alpha`: Double, scale for the negative factor.
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
+
+**Scala example:**
+```scala
+import com.intel.analytics.bigdl.nn.keras.{Sequential, ELU}
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor._
+
+val model = Sequential[Float]()
+model.add(ELU(inputShape = Shape(3)))
+val input = Tensor[Float](2, 3).randn()
+val output = model.forward(input)
+```
+Input is:
+```scala
+input: com.intel.analytics.bigdl.tensor.Tensor[Float] =
+-0.13405465	0.05160992	-1.4711418
+1.5808829	-1.3145303	0.6709266
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x3]
+```
+Output is:
+```scala
+output: com.intel.analytics.bigdl.nn.abstractnn.Activity =
+-0.1254577	0.05160992	-0.77033687
+1.5808829	-0.73139954	0.6709266
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x3]
+```
+
+**Python example:**
+```python
+import numpy as np
+from bigdl.nn.keras.topology import Sequential
+from bigdl.nn.keras.layer import ELU
+
+model = Sequential()
+model.add(ELU(1.2, input_shape=(3, )))
```

Comment:
Change python parameter to 1.0 as well.....
Always scala parameters = python parameters